supportedBackends/sfs/src/main/scala/cromwell/backend/impl/sfs/config/ConfigAsyncJobExecutionActor.scala

```diff
@@ -304,7 +306,11 @@ class DispatchedConfigAsyncJobExecutionActor(override val standardParams: Standa
           jobLogger.error(s"Return file not found after ${exitCodeTimeout.getOrElse(backupError)} seconds, assuming external kill")
 
           val returnCodeTemp = jobPaths.returnCode.plusExt("kill")
-          returnCodeTemp.write(s"$SIGTERM${System.lineSeparator}")
+          // If SIGTERM, SIGKILL or SIGINT codes are used, cromwell will assume the job has been aborted by cromwell.
+          // And it will therefore NOT retry the job. Which makes perfect sense. Best not to change that in the
+          // StandardAsyncExecutionActor code, but give a nonsensical return code here.
+          val exitValue = if (retryExternallyKilledJobs) Int.MaxValue else SIGKILL
```

Comment:
Not overloading `SIGKILL` et al sounds sensible but I'd still suggest staying without the usual exit code range. Per http://tldp.org/LDP/abs/html/exitcodes.html:
> An exit value greater than 255 returns an exit code modulo 256

In this case, `2,147,483,647 mod 256 = 255`(!!)

The same page has this suggestion (and this seems sensible to me too):
> The author of this document proposes restricting user-defined exit codes to the range 64 - 113